### ISSUE TITLE
Lumberjack CSS is only applied when GridFieldSiteTreeAddNewButton is present

### ISSUE THREAD

**Issue by bcairns** [silverstripe/silverstripe-lumberjack#93]
If you don't use the GridFieldSiteTreeAddNewButton component, lumberjack.css does not get added as a Requirement, resulting in an unstyled State column:

![image](https://user-images.githubusercontent.com/638372/65265367-04ca5d80-dace-11e9-855a-5639773f2a87.png)

Including GridFieldSiteTreeState should add lumberjack.css as a Requirement, since it uses classes from that file.

A workaround for this issue is to add this to your config:

```
SilverStripe\Admin\LeftAndMain:
  extra_requirements_css:
    - silverstripe/lumberjack:css/lumberjack.css
```

**Comment by robbieaverill** [comment]
Would you mind making a pull request for that replacement? `Requirements` calls don't work in SilverStripe 4 with dynamically loaded sections of the CMS

**Comment by bcairns** [comment]
Sorry, are you saying the solution should be adding the above snippet to lumberjack's YML?  (instead of calling Requirements::css() in code somewhere?)

Should we do the same thing for Lumberjack's Gridfield.js then?  And then remove the two existing Requirements calls from GridFieldSiteTreeAddNewButton?

**Comment by NightJar** [comment]
> Would you mind making a pull request for that replacement? `Requirements` calls don't work in SilverStripe 4 with dynamically loaded sections of the CMS

Sorry, could you expand on this a little please? I was under the impression this still worked, unless one is viewing an entirely React based page (i.e. AssetAdmin). But Lumberjack affects PagesAdmin, so I think `jQuery.onDemand` still operates fine.

**Comment by robbieaverill** [comment]
Yeah, it’s a core bug but this is why we usually put requirements in YAML for SS4 upgraded modules

**Comment by NightJar** [comment]
oophf. Cool, good to know.
In this case then, yes @bcairns - please use the `LeftAndMain.extra_requirements_css` config rather than PHP's `Requirements` :)

**Comment by robbieaverill** [comment]
Fixed in 2.0/master with 0e26ab2db00203096e9158395d112b46715b7f91, thanks @bcairns
